### ISSUE TITLE
Highlight breaks when performing this kind of tuple unpacking

### ISSUE THREAD

**Issue by giopaglia** [JuliaEditorSupport/Julia-sublime#98]
Hi! Thanks for the package! I'm a huge fan.
I've noticed the syntax highlighting breaks when you define functions like
```julia
foo((X,Y)) = X+Y
```
I don't know the exact name of this language feature, but it's some sort of tuple unpacking that happens in the function's header.
I'm on Julia 1.5.4. Here are some valid definitions that unveil the highlighting problem
```julia
foo2(a, (X,Y), b) = X+Y
foo3(a::Integer, (X,Y)::Tuple{Integer}, b::Integer, c::Integer) = X+Y
foo3(a::Integer, (X,Y)::NTuple{2,Integer}, b::Integer, c::Integer) = X+Y
```

Thanks so much! =)

**Comment by randy3k** [comment]
The latest version only works in Sublime Text 4 and here is the screenshot of ST 4.
<img width="740" alt="Screen Shot 2021-03-29 at 5 54 38 PM" src="https://user-images.githubusercontent.com/1690993/112917618-e0364500-90b7-11eb-9700-c224a1c80538.png">

**Comment by giopaglia** [comment]
Oh I see, thanks again.

**Comment by rasmushenningsson** [comment]
I'm getting a similar problem in the latest version (Sublime Text build 4107 and Julia plugin v1.0.2)
![highlight](https://user-images.githubusercontent.com/16546530/119316665-84330b80-bc77-11eb-82ee-7fa9b05bc5db.png)

**Comment by randy3k** [comment]
@rasmushenningsson Thanks for reporting, I could reproduce it. Working on a patch now.

**Comment by randy3k** [comment]
fixed in v1.0.3, see 3d47f0c.

**Comment by rasmushenningsson** [comment]
Thanks for the super quick fix. It works great.
